server/cli/launch.py

```diff
@@ -317,6 +325,9 @@ def launch(
         except DatasetAccessError as e:
             raise click.ClickException(str(e))
 
+        if experimental_enable_reembedding and matrix_data_loader.matrix_data_type() != MatrixDataType.H5AD:
```

Comment:
Not important for this PR, but I think it would be quite easy to get this to work with TileDb.   Anndata has a constructor that takes in X, obs and var arrays.   So, it's just a matter of how you initialize the temporary AnnData object before you compute the umap.

Reply:
Agree - I have an alternate thought on how to do it, but I agree it should be easily in principle (I'm not sure we want or need to use ScanPy/Anndata - I think we just integrate the underlying embedding module Scanpy wraps).